packages/model-viewer/src/test/features/scene-graph/material-spec.ts

```diff
@@ -117,4 +120,92 @@ suite('scene-graph/material', () => {
       expect(threeTexture.uuid).to.be.equal(newUUID);
     });
   });
+
+  suite('Material properties', () => {
+    let element: ModelViewerElement;
+
+    setup(async () => {
+      element = new ModelViewerElement();
+    });
+
+    teardown(() => {
+      document.body.removeChild(element);
+    });
+
+    const loadModel = async (path: string) => {
+      element.src = path;
+      document.body.insertBefore(element, document.body.firstChild);
+      await waitForEvent(element, 'load');
+    };
+
+    test('test alpha cutoff expect disabled by default', async () => {
+      await loadModel(HELMET_GLB_PATH);
+      // Disabled cutoff value hack see:
+      // https://github.com/google/model-viewer/blob/e3a000111980f5cf018c61dec8695463e0d843a0/packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts#L210
+      expect(element.model!.materials[0].getAlphaCutoff())
+          .to.be.equal(ALPHA_CUTOFF_OPAQUE);
+    });
+
+    test('test alpha cutoff expect valid value as default', async () => {
+      await loadModel(ALPHA_BLEND_MODE_TEST);
+      expect(element.model!.materials[2].getAlphaCutoff()).to.be.equal(0.25);
+    });
+
+    test('test alpha cutoff test setting and getting', async () => {
+      await loadModel(ALPHA_BLEND_MODE_TEST);
+      element.model!.materials[2].setAlphaCutoff(0.0);
```

Comment:
This is kind of a funny test because if you set a MASK mode material to a cutoff of 0.0, you end up getting a BLEND material, right? I don't think there's much we can do about that until three fixes things upstream, but maybe test a non-zero value?

Reply:
Really it just needs to read the same value that was set, but in the crazy world, I see your point.